src/main/java/frc/robot/HardwareMap.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot;
+
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.SpeedControllerGroup;
+import edu.wpi.first.wpilibj.XboxController;
+
+public class HardwareMap {
+
+    public class ShooterHardware{
+        public CANSparkMax leftShooter = new CANSparkMax(16, MotorType.kBrushless);
```

Comment:
move the construction of objects into the constructor of shooter hardware. (the new CANSparkmax part)